Gemfile.lock

```diff
@@ -25,10 +25,12 @@ GEM
     ethon (0.12.0)
       ffi (>= 1.3.0)
     eventmachine (1.2.7)
+    eventmachine (1.2.7-x64-mingw32)
```

Comment:
This is weird. Why are there 2 eventmachine dependencies?

Reply:
So this would be windows one? It should be fine, but if the build/site breaks, I would look at this as the first culprit.

Reply:
I have Gemfile.lock exempt from commit, I believe

Reply:
Hmm, maybe we should exempt it here, too, later on.